app/test_main.py

```diff
@@ -1,3 +1,25 @@
+import pytest
 from app.main import get_human_age
```

Comment:
fix imports according to PEP8
![Screenshot from 2023-10-29 13-54-16](https://github.com/mate-academy/py-cat-and-dog-years/assets/122517203/2ad1574c-de15-4e25-87af-ce404e04182b)